R/estimateParams.R

```diff
@@ -109,7 +109,7 @@ haz <- function(transition, t, trans) {
 haz.ExponentialTransition <- function(transition, t, trans) {
   # params (in this order): h01, h02, h12.
   params <- unlist(transition$hazards, use.names = FALSE)
-  params[trans]
+  rep(params[trans], length(t))
```

Comment:
seems something was fixed here - please add a corresponding unit test for this function

Reply:
Actually, it seems like this causes some problem with `negLogLik()`. Similar to `haz.WeibullTransition`, I want `haz.ExponentialTransition` to be able to take a vector as argument `t`. However, if I let it return `rep(params[trans], length(t))`, this does not work with the current implementation of `negLogLik()`. I tried using `apply` instead within `negLogLik()`, but this seems to increase runtime by a lot. Do you have an idea?

Reply:
Nevermind I think I found a solution :)